docs/data-sources/notification_destinations.md

```diff
@@ -0,0 +1,63 @@
+---
+subcategory: "Workspace"
+---
+# databricks_notification_destinations Data Source
+
+-> **NOTE** Ensure that the notification destinations you are attempting to retrieve have been created and are accessible within your workspace.
```

Comment:
Nit: we don't need `**NOTE**` with `->` it will be added automatically.

We also don't need this warning, because we don't fail on the empty list anymore.